src/main/webAuthnClient.ts

```diff
@@ -94,6 +111,42 @@ export default class WebAuthnClient {
     }
   }
 
+  resetPasskeys(params: ResetPasskeysParams): Promise<void> {
+    if (window.PublicKeyCredential) {
```

Comment:
Juste une remarque. Il me semble que le SDK Core est sensé être utilisable sur serveur Nodejs par exemple. Or en Nodejs `window` est `undifined`, donc on aurait une erreur ici.
Je suis sûr que c'est fait comme ça à d'autres endroits et d'ailleurs il faudrait peut-être à l'occasion prévoir un audit de compatibilité Nodejs si c'est quelque chose auquel on tient.

Dans ce sens il faudrait faire `if(window && 'PublicKeyCredential' in window)` par exemple.